jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/JKubeProjectUtilTest.java

```diff
@@ -46,16 +54,14 @@ public void hasDependencyWithGroupIdWithNulls() {
   }
 
   @Test
-  public void hasDependencyWithGroupIdWithDependency(@Mocked JavaProject project) {
+  public void hasDependencyWithGroupIdWithDependency() {
     // Given
     // @formatter:off
```

Comment:
Please remove these `// @formatter: off` and `// @formatter: on` statements. These were used earlier to instruct IDE to not autoformat `new Expectations {{` block. Since we've removed it now, it shouldn't be required anymore.

Reply:
Sure,Now removed from all the classes!